tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -621,13 +603,19 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
         } else if flags.action.kind == "nodejs:6" || flags.action.kind == "nodejs:6.0" ||
             flags.action.kind == "nodejs:6.0.0" {
             action.Exec.Kind = "nodejs:6"
-        } else if matched, _ := regexp.MatchString(".swift$", stat.Name()); matched {
+        } else if len(flags.action.kind) > 0 {
```

Comment:
There are multiple runtimes for Swift, and NodeJs. The --kind argument is needed for those to specify which version to use. Where as Python, and Java only have one runtime a piece.